WooCommerce/Classes/ViewRelated/ReusableViews/SwiftUI Components/AdaptiveModalContainer.swift

```diff
@@ -88,3 +90,37 @@ struct AdaptiveModalContainer<PrimaryView: View, SecondaryView: View>: View {
         }
     }
 }
+
+enum AdaptiveModalContainerPresentationStyle {
+    case modalOnModal
+    case sideBySide
+}
+
+struct AdaptiveModalContainerPresentationStyleKey: EnvironmentKey {
+    static let defaultValue: AdaptiveModalContainerPresentationStyle = .modalOnModal
+}
+
+extension EnvironmentValues {
+    var adaptiveModalContainerPresentationStyle: AdaptiveModalContainerPresentationStyle {
+        get { self[AdaptiveModalContainerPresentationStyleKey.self] }
+        set { self[AdaptiveModalContainerPresentationStyleKey.self] = newValue }
+    }
+}
+
+
+class AdaptiveModalContainerPresentationContext: ObservableObject, Equatable {
+    static func == (lhs: AdaptiveModalContainerPresentationContext, rhs: AdaptiveModalContainerPresentationContext) -> Bool {
+        return lhs.presentationStyle == rhs.presentationStyle
+    }
```

Comment:
Great spot! I think I used this with an attempt to use `@EnvironmentObject`, then didn't notice that it wasn't part of the eventual `@Environment` solution that I went with (as it's nicer.)

Fixed in efaed38

> Should we move the Equatable conformance to an extension? Perhaps the PresentationStyle enum as well.

Since it's unused, we can just delete both. The `AdaptiveModalContainerPresentationStyle` still exists and is the one that actually gets used, but it needs to stay stand-alone.